users/views.py

```diff
@@ -26,6 +26,12 @@
 from rest_framework.permissions import AllowAny
 from rest_framework.throttling import AnonRateThrottle
 from rest_framework_simplejwt.views import TokenObtainPairView
+from rest_framework.permissions import AllowAny
+from rest_framework.throttling import ScopedRateThrottle
+from django.utils.http import urlsafe_base64_encode
+from django.utils.encoding import force_bytes
+from .serializers import ResendEmailSerializer
+from .tokens import email_verification_token
```

Comment:
You're importing AllowAny twice (lines 26 and 29); remove the duplicate import for clarity.
Imports are not grouped according to convention (standard library, third-party, local); consider organizing them for readability.